lms/routes.py

```diff
@@ -13,8 +13,9 @@ def includeme(config):
 
     config.add_route("config_xml", "/config_xml")
     config.add_route(
-        "module_item_configurations",
-        "/module_item_configurations",
+        "configure_assignment",
+        "/assignment",
+        request_method="POST",
```

Comment:
This is a private URL that our own frontend code posts to when creating assignments (the backend passes the URL to the frontend in the JS config), so we can change it at will. I just changed it to `/assignment` which seems RESTful: `POST`'ing to `/assignment` to create an assignment. Added `request_method="POST"` to clarify that the route is only used for `POST`'ing to (otherwise I'd have to change the route name from `"configure_assignment"` to just `"assignment"`).

Reply:
We might even want to rename the route `"create_assignment"` instead of `"configure_assignment"`

Reply:
Note that this really is just for creating new assignments. You cannot edit existing assignments in the LMS app. At least not in non-Canvas LMS's. In Canvas you can, but in Canvas this route isn't used at all and we submit assignment creates and updates to Canvas rather than to ourselves.